### ISSUE TITLE
Any way to stop or pause autoplay?

### ISSUE THREAD

**Issue by NickWoodward** [rcbyr/keen-slider#430]
I'm using the autoplay demo here: https://codesandbox.io/p/sandbox/vigilant-jasper-pvpkp4

Was just wondering if there was any way to prevent the auto play from continuing on a button click? or even on drag. All my attempts seemed to have been ignored :/

**Comment by mgoriachev** [comment]
@NickWoodward Hello! I had a similar problem. Were you able to solve it?

**Comment by NickWoodward** [comment]
> [@NickWoodward](https://github.com/NickWoodward) Hello! I had a similar problem. Were you able to solve it?

no, not if it's the continuous autoplay. if it's not continuous/smooth, this will do it: https://codesandbox.io/p/sandbox/keen-slider-autoplay-react-e7msi?file=%2Fsrc%2FApp.js
